views/me.hbs

```diff
@@ -0,0 +1,5 @@
+<script>
+// Get the cached user.
+const me = localStorage.myGitHub;
+window.location.href = `/?username=${me}`; 
```

Comment:
It's better to add a check for the existence of the GitHub login (to avoid `/?username=undefined`).

Reply:
Good catch. I'll fix it in the next revision.